make_profiler/parser.py

```diff
@@ -61,7 +61,7 @@ def parse_target(token):
                     sorted(deps.strip().split()) if deps else [],
                     sorted(order_deps.strip().split()) if order_deps else []
                 ],
-                'docs': docstring.strip().strip('#').strip() if docstring else target,
```

Comment:
this will leak into the SVG, we probably want target name on hover rather than word '[EMPTY]'. Probably will be best to put just empty string `''` here, and on SVG generation format the title text as `targetname [: docstring if present]`

Reply:
Should I add a check for dependent targets that are existing as separate files in data directory?